etc/config/settings.sh

```diff
@@ -170,7 +170,10 @@ export FOAM_JOB_DIR=$WM_PROJECT_INST_DIR/jobControl
 # wmake configuration
 export WM_DIR=$WM_PROJECT_DIR/wmake
 export WM_LINK_LANGUAGE=c++
-export WM_OPTIONS=$WM_ARCH$WM_COMPILER$WM_PRECISION_OPTION$WM_COMPILE_OPTION
+WM_SUFIX=${WM_GPU_ARCH^^}
```

Comment:
It would be nice to have a comment explaining what will be the final result of these transformations.